tests/search.spec.ts

```diff
@@ -41,7 +41,7 @@ test("search", async ({ page }) => {
   await expect(firstSearchResult.locator('[data-cy="family-metadata-category"]')).toBeVisible();
   await expect(firstSearchResult.locator('[data-cy="family-metadata-year"]')).toBeVisible();
   await expect(firstSearchResult.locator('[data-cy="family-description"]')).toBeVisible();
-  await expect(firstSearchResult.locator('[data-cy="country-link"]')).toBeVisible();
+  await expect(firstSearchResult.locator('[data-cy="country-link"]')).toBeVisible({ timeout: 10000 });
```

Comment:
question: should we be doing this? I know it fixes the test - but it's a cludge to a wider issue. 

If we are just looking to get the e2e tests less stable, could we perhaps create an issue and TODO to tidy this up in future?